tests/cases/distributed/optimizer/filter_push_down.sql

```diff
@@ -1,15 +0,0 @@
-CREATE TABLE integers(i INTEGER, j BIGINT TIME INDEX);
```

Comment:
Why remove this file

Reply:
It's moved to the "common" dir, shared by standalone and distributed.